src/Common/Stopwatch.cpp

```diff
@@ -7,8 +7,12 @@ StopwatchRUsage::Timestamp StopwatchRUsage::Timestamp::current()
 
     ::rusage rusage {};
 #if !defined(__APPLE__)
+#if defined(OS_SUNOS)
+    ::getrusage(RUSAGE_LWP, &rusage);
```

Comment:
For reference, LWP refers to a "light-weight process", and is functionally analogous to a thread on modern illumos systems.